pallets/claims/src/lib.rs

```diff
@@ -230,11 +230,10 @@ pub mod pallet {
 	pub(super) type ClaimedAmounts<T: Config> =
 		StorageMap<_, Blake2_128Concat, T::AccountId, T::Balance, ValueQuery>;
 
-	/// Map of root hashes that correspond to lists of reward claim amounts per account.
+	/// Root hash that correspond to lists of reward claim amounts per account.
 	#[pallet::storage]
 	#[pallet::getter(fn get_root_hash)]
-	pub(super) type RootHashes<T: Config> =
-		StorageMap<_, Blake2_128Concat, T::Hash, bool, ValueQuery>;
+	pub(super) type RootHash<T: Config> = StorageValue<_, T::Hash, OptionQuery>;
```

Comment:
The old storage is only of size 1, so I think a cleanup migration is an overkill

Reply:
If it was a difficult migration I'd agree, but since it's a one-liner to nuke the storage I think we should just do it